src/simularium/index.ts

```diff
@@ -5,6 +5,8 @@ export type {
     TrajectoryFileInfo,
     EncodedTypeMapping,
     SimulariumFileFormat,
+    SelectionEntry,
+    ColorChanges,
```

Comment:
I think the types are already defined in SelectionInterface (the import below this line).  Could you just get it from there and skip adding it to types.ts?

Reply:
Yes I think so!

The comments from Megan that I was trying to address when I created the error, had to do with getting these types exported from the right places. I'm honestly not sure what best practice would be... declaring them in SelectionInterface or types.ts?

Seems like we can resolve the errors with either approach.

Reply:
so right now with this PR they are declared in two places. It would be better if they are defined only once.

Reply:
Ok, best to keep it in SelectionInterface I think.